app/views/people/index.html.erb

```diff
@@ -0,0 +1,13 @@
+
+<% @people.each do |person| %>
+  <% tasks = [] %>
+  <% person.tasks.each do |task| %>
+    <% unless task[:completed_date] %>
+    <% tasks.push task %>
+    <% end %>
+  <% end %>
+  <% id = person[:favorite_color] %>
+  <h3 id="people"> <%= link_to "#{person[:name]}", "/people/#{person[:id]}", id: id %> has <%= tasks.length %> task(s) unaccounted for. </h3>
```

Comment:
Here is a place where you *do* have multiple HTML elements with the same ID. This can cause strange behavior later so here I'd use a class rather than and ID. You only want to use the ID when it will always be unique.